core/session.ts

```diff
@@ -293,6 +317,18 @@ export class Session {
     return newTest;
   }
 
+  public declare(fullyQualifiedDataset: { schema: string; name: string }): Declaration {
+    const { name } = fullyQualifiedDataset;
+    this.checkActionNameIsUnused(name);
+    const declaration = new Declaration();
+    declaration.proto.name = name;
```

Comment:
nit - I'd switch lines 324 and 325 because going forward name will mimic target.

Reply:
Done!